src/app/pages/about/about.component.html

```diff
@@ -51,7 +51,7 @@
       <mat-card-content>
         <mat-nav-list>
           <mat-list-item *ngFor="let contributor of allContributors">
-            <img matListAvatar src="{{contributor.avatarUrl}}&size=40" srcset="{{contributor.avatarUrl}}&size=80 2x, {{contributor.avatarUrl}}&size=160 4x">
+            <img matListAvatar src="{{contributor.avatarUrl}}&size=40" srcset="{{contributor.avatarUrl}}&size=80 2x, {{contributor.avatarUrl}}&size=160 4x" alt="{{contributor.username}} GitHub avatar">
```

Comment:
```suggestion
            <img matListAvatar src="{{contributor.avatarUrl}}&size=40" srcset="{{contributor.avatarUrl}}&size=80 2x, {{contributor.avatarUrl}}&size=160 4x" alt="">
```

Reply:
Since this is a clickable element in a list I think it should have an alt tag, even GitHub themselves give your avatar on this page `alt="@helloworld12321"`.

What do you think about matching their behavior and doing this: 
```suggestion
            <img matListAvatar src="{{contributor.avatarUrl}}&size=40" srcset="{{contributor.avatarUrl}}&size=80 2x, {{contributor.avatarUrl}}&size=160 4x" alt="@{{contributor.username}}">
```